scripts/facts.runs/format_facts.py

```diff
@@ -23,11 +23,24 @@
 args = parser.parse_args()
 
 # Access the lists using the argument names
-pulse_years = args.pulse_years
-gases = args.gases
 facts_dir = args.facts_repo[0]
 gmsl_pulse = args.gmsl_pulse[0]
 
+if args.pulse_years:
+    pulse_years = list(map(int, args.pulse_years))
+else:
+    print("No pulse years specified")
```

Comment:
Will L29 in `facts_runs.sh` fail if no pulse_years and gases are specified? e.g. will we reach this `else` block?

Reply:
If no pulse years and gases are specified then args.pulse_years = None and we go to the else statement